src/python/pants/backend/docker/util_rules/docker_build_context_test.py

```diff
@@ -393,7 +393,8 @@ def test_packaged_pex_environment(rule_runner: RuleRunner) -> None:
                 """
               docker_environment(
                 name="python_38",
-                image="python:3.8",
+                image="python:3.8-buster@sha256:bc4b9fb034a871b285bea5418cedfcaa9d2ab5590fb5fb6f0c42aaebb2e2c911",
+                platform="linux_x86_64",
```

Comment:
The sha256 digest is specific to a particular platform, so we need to be specific here, or else running on arm64 (e.g. M1 Mac) fails because the x86-64 image doesn't match the default (`linux_arm64`).